Running Node.js sample fails (dist directory doesn't exist)

I'm trying to run the [Node.js samples](https://github.com/tigerbeetle/tigerbeetle/tree/db706e09bc7a8f73242bc4d855a878d127d97812/src/clients/node/samples/two-phase) but ran into the following error:

```
❯ node main.js                
node:internal/modules/cjs/loader:446
      throw err;
      ^

Error: Cannot find module 'tigerbeetle/src/clients/node/samples/two-phase/node_modules/tigerbeetle-node/dist/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:438:19)
    at Module._findPath (node:internal/modules/cjs/loader:680:18)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1063:27)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:121:18)
    at Object.<anonymous> (tigerbeetle/src/clients/node/samples/two-phase/main.js:8:5)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32) {
  code: 'MODULE_NOT_FOUND',
  path: 'tigerbeetle/src/clients/node/samples/two-phase/node_modules/tigerbeetle-node/package.json',
  requestPath: 'tigerbeetle-node'
}

Node.js v18.17.1
```

It looks like installing `tigerbeetle-node` as specified in the instructions doesn't build the files (the `dist` directory doesn't exist) that the `package.json` tells it to expect.

--- a/src/clients/node/README.md
+++ b/src/clients/node/README.md
@@ -21,7 +21,8 @@ First, create a directory for your project and `cd` into the directory.
 Then, install the TigerBeetle client:
 
 ```console
-npm install tigerbeetle-node
+npm install --prefix ../../
+npm install
 ```
 
 Now, create `main.js` and copy this into it:

--- a/src/clients/node/docs.zig
+++ b/src/clients/node/docs.zig
@@ -87,7 +87,10 @@ pub const NodeDocs = Docs{
     \\console.log("Import ok!");
     ,
 
-    .install_commands = "npm install tigerbeetle-node",
+    .install_commands =
+    \\npm install --prefix ../../
+    \\npm install
+    ,
     .build_commands =
     \\npm install typescript @types/node
     \\npx tsc --allowJs --noEmit main.js

--- a/src/clients/node/samples/basic/README.md
+++ b/src/clients/node/samples/basic/README.md
@@ -18,7 +18,8 @@ First, clone this repo and `cd` into `tigerbeetle/src/clients/node/samples/basic
 Then, install the TigerBeetle client:
 
 ```console
-npm install tigerbeetle-node
+npm install --prefix ../../
+npm install
 ```
 
 ## Start the TigerBeetle server

--- a/src/clients/node/samples/two-phase-many/README.md
+++ b/src/clients/node/samples/two-phase-many/README.md
@@ -18,7 +18,8 @@ First, clone this repo and `cd` into `tigerbeetle/src/clients/node/samples/two-p
 Then, install the TigerBeetle client:
 
 ```console
-npm install tigerbeetle-node
+npm install --prefix ../../
+npm install
 ```
 
 ## Start the TigerBeetle server

--- a/src/clients/node/samples/two-phase/README.md
+++ b/src/clients/node/samples/two-phase/README.md
@@ -18,7 +18,8 @@ First, clone this repo and `cd` into `tigerbeetle/src/clients/node/samples/two-p
 Then, install the TigerBeetle client:
 
 ```console
-npm install tigerbeetle-node
+npm install --prefix ../../
+npm install
 ```
 
 ## Start the TigerBeetle server

